new_samples/worker/worker.go

```diff
@@ -0,0 +1,86 @@
+// Package worker implements a Cadence worker with basic configurations.
+package worker
+
+import (
+	"github.com/uber-common/cadence-samples/new_samples/workflows"
+	"github.com/uber-go/tally"
+	apiv1 "github.com/uber/cadence-idl/go/proto/api/v1"
+	"go.uber.org/cadence/.gen/go/cadence/workflowserviceclient"
+	"go.uber.org/cadence/activity"
+	"go.uber.org/cadence/compatibility"
+	"go.uber.org/cadence/worker"
+	"go.uber.org/cadence/workflow"
+	"go.uber.org/yarpc"
+	"go.uber.org/yarpc/transport/grpc"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+const (
+	HostPort       = "127.0.0.1:7833"
+	Domain         = "cadence-samples-domain"
+	TaskListName   = "cadence-samples-worker"
+	ClientName     = "cadence-samples-worker"
+	CadenceService = "cadence-frontend"
+)
+
+// StartWorker creates and starts a basic Cadence worker.
+func StartWorker() {
+	// TaskListName identifies set of client workflows, activities, and workers.
+	// It could be your group or client or application name.
```

Comment:
accurate but probably belongs below, or on `TaskListName` itself

Reply:
I put this comment on the constant definition itself above.